README.md

```diff
@@ -50,17 +50,52 @@ And it won't change your existing workflow.
        alt="invocation results">
 </p>
 
+### Example
```

Comment:
Let's remove this section for now:
1. I will make screenshots with the same style as "Running"
2. I will have to think about some cool examples to showcase

Reply:
I have removed this section and I will push the PR again.  
Do check and let me know if this is fine.